vulpes/fixes/framerate_dependent_timers.cpp

```diff
@@ -130,14 +130,57 @@ void revert_console_fix() {
     DEL_CALLBACK(EVENT_TICK, fade_console);
 }
 
+/*
+    Contrails are updated each frame, they are updated based on how much time
+    has passed between this and the last frame. Sometimes it updates a little
+    finnicky though and the contrails have visible distortions.
+
+    Turns out that contrails look fine if you just only update them every tick.
+
+    When do this, we do supply the time that passes between two ticks as
+    one of the arguments to the function in our assembly. The same value
+    that would be supplied at 30fps, where the contrails function properly.
```

Comment:
Would you try writing that again?

Reply:
Is this better?

> In the stock game, contrails are updated every frame. They update based on how much time has passed between the current frame and the previous frame. This calculation was designed with the expectation that the game's frame rate would match the tick rate (30 fps). Because of this, the contrails can have visible distortions at higher frame rates (ex: the Banshee wing contrails having little hiccups when they should be smooth curves).
> 
> This issue does not happen if the contrails are updated every tick instead of every frame. We can fix this by hooking into the contrail code and feeding it the time between two ticks instead of the time between two frames.
>